packages/editor-ui/src/components/Node.vue

```diff
@@ -161,6 +171,49 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 				this.setSubtitle();
 			}
 		},
+		nodePosition: {
+			handler() {
+				// Re-render the helping tooltip for trigger node if its position is changed
+				if (this.showWebhookNodeTooltip) {
+					this.showWebhookNodeTooltip = false;
+					setTimeout(() => {
+						this.showWebhookNodeTooltip = true;
+					}, 100);
+				}
+			},
+			immediate: true,
+			deep: true,
+		},
+		nodeOffsetPosition: {
+			handler() {
+				// Re-render the helping tooltip for trigger node if the workflow is zoomed-out/in
+				if (this.showWebhookNodeTooltip) {
+					this.showWebhookNodeTooltip = false;
+					setTimeout(() => {
+						this.showWebhookNodeTooltip = true;
+					}, 100);
+				}
+			},
+			immediate: false,
+			deep: true,
+		},
+		workflowRunning: {
+			handler(bool) {
+				// Show helping tooltip for trigger node if:
+				// #workflow is runnung (bool)
+				// #if it is a trigger node (this.isWebhookNode)
+				// #if it is only one trigger node in the WF (this.isSingleWebhookNode)
+				// #if the node is not disabled (!this.data.disabled)
+				if (bool && this.isWebhookNode() && this.isSingleWebhookNode() && !this.data.disabled) {
```

Comment:
these comments are not helpful.. it's preferrable to minimize comments as they go stale and irrelevant quickly. instead try to have the code naming (function names, variable names) express what the code does...
in this case rename `bool` to something more helpful and delete the comments  (maybe `isWorkflowRunning`)

Reply:
The comments are removed and the bool is refactored to isWorkflowRunning. :)